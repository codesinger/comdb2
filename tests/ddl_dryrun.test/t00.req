insert into t1 values(1, x'abcdefabcdef', x'abcdefabcdef',1)
insert into t1 values(2, x'abcdefabcdef', x'abcdefabcdef',2)
insert into t1 values(3, x'abcdefabcdef', x'abcdefabcdef',3)
select * from t1
select * from t1 group by id
create procedure sptest version 'test' {}
