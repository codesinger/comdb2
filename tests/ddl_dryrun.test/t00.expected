(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(id=1, data=x'abcdefabcdef', inline_data=x'abcdefabcdef', u=1)
(id=2, data=x'abcdefabcdef', inline_data=x'abcdefabcdef', u=2)
(id=3, data=x'abcdefabcdef', inline_data=x'abcdefabcdef', u=3)
(id=1, data=x'abcdefabcdef', inline_data=x'abcdefabcdef', u=1)
(id=2, data=x'abcdefabcdef', inline_data=x'abcdefabcdef', u=2)
(id=3, data=x'abcdefabcdef', inline_data=x'abcdefabcdef', u=3)
(version='test')
